components/sample/keras/train_classifier/component.yaml

```diff
@@ -0,0 +1,27 @@
+name: Keras - Train classifier
+description: Trains classifier using Keras sequential model
+inputs:
+  - {name: Training set features path, type: {GcsUri: TSV}, description: 'Local or GCS path to the training set features table.'}
```

Comment:
Can we have default values for all of these?

Reply:
I think no. The default should be added when it covers majority of the use cases. For these data inputs that's not the case. There is no reasonable default value for the training set or even the model config.